blackbox/docs/sql/reference/copy_to.txt

```diff
@@ -22,11 +22,12 @@ Description
 ===========
 
 The ``COPY TO`` command exports the contents of a table to one or more
-files in the given directory, named to prevent filename conflicts. Each
-node of the cluster which contains a shard of the table will export the
-contents of its shards.
+files into a given directory with unique filenames. Each node with at least
+one shard of the table will export the contents of these onto their local disk.
```

Comment:
you say at least one shard and then "these" which doesn't easily refer to shards